pages/lessons/projects/4.mdx

```diff
@@ -12,15 +12,24 @@ import { LessonHeader } from '../../../components/mdx/LessonHeader'
   discussionUrl="https://developerdao.peeranha.io/discussions/1372/testing-solidity-contracts"
 />
 
-In this lesson, we'll be adding automated tests for our smart contract created
-in Lesson 3.
+## About this lesson
 
-Here you'll:
+Welcome to this lesson where we’ll be adding automated tests for our *TierNFT* smart contract created in the previous tutorial. If you haven’t completed it already, we have a fast-track guide below to that project set-up - assuming you’re already confident with Solidity and a professional development environment. If not, we’d recommend completing the earlier tutorials in this track. 
+
+Over here, you’ll learn about the benefits of writing automated tests, general testing concepts and best practices. You’ll walk away with many practical tests for smart contracts and nibble on some food for thought about test-driven development. 
+
+As you dive in to the lesson, you'll come across some thought-provoking checkpoint questions designed to test your existing knowledge, measure how well you're absorbing the fresh content, and maybe ask you to predict next steps in the lesson. And to wrap up, a final quiz awaits. To complete this adventure might take anywhere from one to six hours. That all depends on your previous experience and how much new ground you need to cover. But enjoy the journey and remember to take regular breaks along the way. We care about your well-being and have even included a nifty pomodoro timer ⌛ in the menu header. Let it remind you to appreciate nature's gifts and "go touch some grass" for a while. 🌱 Embrace the process, and let's set sail!
+
+<SideDrawer buttonText="Warm-up Questions" title="Check out Your Previous Knowledge">
+      <Question question="lesson-4/1-intro/Q1" />
+      <Question question="lesson-4/1-intro/Q2" />
+      <Question question="lesson-4/1-intro/Q3" />
+      <Question question="lesson-4/1-intro/Q4" />
+      <Question question="lesson-4/1-intro/Q5" />
+</SideDrawer> 
+
+PART 1
```

Comment:
are these `PART #`'s for the sake of your review process or are they intended to remain in the text? based on past PRs, i'm assuming the former and that they need removing prior to shipping.